src/dmd/astbase.d

```diff
@@ -4085,6 +4085,24 @@ struct ASTBase
         }
     }
 
+    extern (C++) class TypeTraits : Type
```

Comment:
Needs an entry in `visitor.d`, too.

Reply:
If i do so i get a conflict with the entry in `transitivevisitor.d`